frontend/src/components/Statssection.tsx

```diff
@@ -0,0 +1,91 @@
+import { Grid } from '@mui/material';
+import { StatsService } from 'src/services/openapi';
+import type { Statistics } from 'src/services/openapi';
+import React, { useEffect, useState } from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+interface statsType {
+  text: string;
+  count: number;
+  last_month_count: number;
+}
+
+const StatsUi = (props: statsType) => {
+  return (
+    <Grid item xs={4}>
+      <span style={{ fontSize: '12px' }}>{props.text}</span> <br />
+      <span style={{ fontSize: '50px', lineHeight: '1em' }}>
+        {' '}
+        {props.count}{' '}  
+      </span>{' '}
+      <br />
+      {
+        <span
+          style={
+            props.last_month_count >= 0 ? { color: 'green' } : { color: 'red' }
+          }
+        >
+          {(props.last_month_count >= 0 ? '+ ' : '') + props.last_month_count}
+        </span>
+      }
+    </Grid>
+  );
+};
+
+const StatsSection = () => {
+  const [data, setData] = useState({
```

Comment:
You should precise the type of `data` to `Statistics` like `useState<Statistics>`